src/components/label/Label.tsx

```diff
@@ -53,7 +64,7 @@ const Label = ({ text, tooltip, infoTooltip, infoIconProps, showOptionalText, hi
     }
 
     return (
-        <StandardLabel labelContent={labelContent} tooltip={tooltip}>
+        <StandardLabel labelContent={labelContent} tooltip={tooltip} labelElProps={labelElProps}>
```

Comment:
What about `<HiddenLabel>` also rendered by this component? Should we also pass it the html label attributes?